.mise/config.toml

```diff
@@ -0,0 +1,21 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+# cspell:ignore pipx
+
+[tools]
+"aqua:sharkdp/fd" = "latest"
+"rust" = { version = "1.82", profile = "default" }
```

Comment:
Is it possible to use the same as the CI here, the "stable" channel?

Reply:
That's probably a good option: I went with the MSRV we documented.

Reply:
Thanks :)